packages/flutter/test/services/lifecycle_test.dart

```diff
@@ -9,18 +9,18 @@ import 'package:flutter_test/flutter_test.dart';
 
 void main() {
   testWidgets('initialLifecycleState is used to init state paused', (WidgetTester tester) async {
-    // The lifecycleState is null initially in tests as there is no
-    // initialLifecycleState.
-    expect(ServicesBinding.instance.lifecycleState, equals(null));
-    // Mock the Window to provide paused as the AppLifecycleState
+    // The lifecycleState is detached initially in tests, since the binding is
```

Comment:
Wait, if it is detached, why doesn't it report that as AppLifecycleState.detached instead of being null?

Reply:
Because the comment is out of date.  It's null initially now (again).